### DISCUSSION TITLE
Time Date Picker set to current time

### DISCUSSION THREAD

**Discussion by muksidin** [Budibase/budibase#11560] [Help] [answered]
Hi, is it possible to change the time in date picker to current time? So i don't need to set the time again.
![Screenshot_1](https://github.com/Budibase/budibase/assets/5665115/04723f0f-2382-44da-9d01-b7b52d462b7d)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @muksidin,

You would likely have to write some JS in this case to automatically set the default to whatever your current time is.

Since your question is based around time, I wasn't sure whether to do an example for only time or a date + time example. I ended up doing both, so hopefully one will help;

**Date + Time**
<img width="1464" alt="Screenshot 2023-08-21 at 08 34 11" src="https://github.com/Budibase/budibase/assets/126772285/8ef1cb3f-acea-4665-8e05-518e419ed6cc">

**Time only**
<img width="1462" alt="Screenshot 2023-08-21 at 08 35 46" src="https://github.com/Budibase/budibase/assets/126772285/d8605aa7-fafc-454c-b5f5-d400a9f6331d">

I hope this helps.

**Reply by muksidin** [reply]
Thanks @ConorWebb96 , this is very helpfull. :)
